clang/test/Driver/darwin-framework-search-paths.c

```diff
@@ -0,0 +1,26 @@
+// UNSUPPORTED: system-windows
+//   Windows is unsupported because we use the Unix path separator `/` in the test.
+
+// Add default directories before running clang to check default
+// search paths.
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: cp -R %S/Inputs/MacOSX15.1.sdk %t/
+// RUN: mkdir -p %t/MacOSX15.1.sdk/System/Library/Frameworks
+// RUN: mkdir -p %t/MacOSX15.1.sdk/System/Library/SubFrameworks
+// RUN: mkdir -p %t/MacOSX15.1.sdk/usr/include
+
+// RUN: %clang -xc %s -target arm64-apple-darwin13.0 -isysroot %t/MacOSX15.1.sdk -E -v 2>&1 | FileCheck --check-prefix=CHECK-C %s
```

Comment:
Should we add a test for DriverKit as well?

Reply:
There is already https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/driverkit-path.c 
Are you thinking of adding to it?

Reply:
Oh, yeah I guess that's where it would go

Reply:
What would you want to check for DriverKit? I naively went ahead and started adding a test, and then I went back and realized I wasn't sure what you wanted me to test.